recipes/pyabel/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "PyAbel" %}
+{% set version = "0.7.4" %}
+{% set md5 = "1f9be2ecdc1ea8424000d62e2d6a4b10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+    - numpy >=1.6
+    - setuptools >=16.0
+    - scipy >=0.14
+    - six >=1.10.0
+  run:
+    - python
+    - numpy >=1.6
+    - setuptools >=16.0
+    - scipy >=0.14
+    - six >=1.10.0
+
+test:
+  imports:
+    - abel
+    - abel.lib
+    - abel.tests
+    - abel.tools
+
+about:
+  home: https://github.com/PyAbel/PyAbel
+  license: MIT
+  license_family: MIT
+  summary: 'A Python package for forward and inverse Abel transforms'
+  description: |
+    PyAbel is a Python package that provides functions for the forward and inverse Abel transforms.
+    The forward Abel transform takes a slice of a cylindrically symmetric 3D object and provides the
+    2D projection of that object. The inverse abel transform takes a 2D projection and reconstructs
+    a slice of the cylindrically symmetric 3D distribution.
+  doc_url: http://pyabel.readthedocs.io/en/latest/readme_link.html
+  dev_url: https://github.com/PyAbel/PyAbel
+
+extra:
+  recipe-maintainers:
+    - DanHickstein
+    - rth
+    - stggh
+    - DhrubajyotiDas
```

Comment:
@rth @stggh @DhrubajyotiDas , are you ok being included as maintainers of PyAbel on [conda-forge]( http://conda-forge.github.io/ )?